ci/release/template/man/d2.1

```diff
@@ -21,9 +21,7 @@
 .Nm
 compiles and renders
 .Ar file.d2
-to
-.Ar file.svg
```

Comment:
Change this back to include the output path. Just add a comma in between the two `.Ar`.

Reply:
Ok I try but I"m not sure what the expected output is here

Reply:
same as `--help`. i actually think it should be `|`